index.d.ts

```diff
@@ -22,7 +22,23 @@ export const Ads: any; // export { default as Ads } from './src/Ads'
 export { AppData, FontFaceDefinition } from './src/Application/index.js';
 export { default as Application } from './src/Application/index.js';
 export const AudioPlayer: any; // export { default as AudioPlayer } from './src/AudioPlayer'
-export const Colors: any; // export { default as Colors } from './src/Colors'
+export type ColorTransform = {
+  alpha(value: number): ColorTransform;
+  hue(value: number): ColorTransform;
+  lightness(value: number): ColorTransform;
+  saturation(value: number): ColorTransform;
+  lighter(value: number): ColorTransform;
+  darker(value: number): ColorTransform;
+  mix(color: string, percentage: number): ColorTransform;
+  get(): number;
+};
+
+export type Colors = {
+  (colorName: string): ColorTransform;
+} & {
+  [colorName: string]: ColorTransform;
+};
+export const Colors: Colors; // export { default as Colors } from './src/Colors'
```

Comment:
Thanks for the PR! I'd happily be willing to help push it forward. Could you please move the type definitions to `src/Colors/index.d.ts` though? This file is meant to just meant to re-export types from each individual SDK module.

Reply:
LGTM thanks!